features/navigation/browse_navigation_history.feature

```diff
@@ -1,22 +0,0 @@
-@javascript
```

Comment:
This file was deleted because it contains "back to grid" checks and "pinbar" feature (both :fire: d)